packages/nextjs/scaffold.config.ts

```diff
@@ -30,7 +30,7 @@ const scaffoldConfig = {
   walletConnectProjectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID || "3a8170812b534d0ff9d794f19a901d64",
 
   // Only show the Burner Wallet when running on hardhat network
-  onlyLocalBurnerWallet: true,
+  onlyLocalBurnerWallet: false,
```

Comment:
I think we want this to be true by default.

I see we have the same _issue_ here as with"pollingInterval": where multiple chains are configured, we can't use the burner wallet in hardhat even if this is true.

I think it's ok. Live dApps shouldn't have hardhat enabled anyway.

Reply:
oops, committed that by accident it seems

Reply:
Ugh, I really need to find a system to prevent that... :sweat_smile: